File: OCBridgeSwift/OCBridgeSwift/ViewController.m
//
//  ViewController.m
//  OCBridgeSwift
//
//  Created by Vicent on 2022/4/12.
//

#import "ViewController.h"
#import "OCBridgeSwift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    TestSwit *ts = [TestSwit new];
    [ts test];
}


@end

File: CarthageDemo/CarthageDemo/ViewController.swift
//
//  ViewController.swift
//  CarthageDemo
//
//  Created by Vicent on 2022/2/27.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let user = User(JSONString: "{\"username\":\"张三\"}")
        print("\(#function):\(String(describing: user?.toJSONString(prettyPrint: true)))")
    }
}

